test_util/cluster_api.py

```diff
@@ -338,6 +340,11 @@ def get_test_app_in_docker(self, ip_per_container):
         return app, test_uuid
 
     def deploy_test_app_and_check(self, app, test_uuid):
+        marathon_user = self.mesos_user
```

Comment:
`mesos_user` doesn't exist anymore, does it? Also, you set `marathon_user` to `default_os_user` below.